### ISSUE TITLE
Create color palette mock-ups (S)

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/water-use-15#98]
***I think we should wait on this task*** until we know if we are using the pie charts: 
A couple of notes here - 

Colorblind friendly becomes more important if we use the pie charts. Otherwise, we aren't doing a side-by-side of any categories, so it is more important to understand which category is _on_ than to see the color differences between categories. 

We'll want a blend between colors that make sense for the categories (e.g., green), aesthetically pleasing combinations, and ability for colorblind to pick out the differences.

**Comment by jread-usgs** [comment]
We have decided to remove the pie charts from the interactive viz

**Comment by aappling-usgs** [comment]
But note that we're still talking about including a static version of the pie charts in a key story

**Comment by lindsaycarr** [comment]
Are colors for background map, states at national view, state color in zoom, boundaries, etc are being chosen? Just wondering if it's my job to do that with #95 or not.

**Comment by jread-usgs** [comment]
I think this is done, but we should create another issue for doing a comprehensive style unification sweep a few days before our next milestone. Revisiting the state borders and zoom styling would be included in that. 
I don't think it is worth getting uber picky about right now since we are still iterating on the look and feel of certain things, including mobile and the category selectors in #89

**Comment by aappling-usgs** [comment]
the colors Jordan has proposed are these:
```
var stateStyle = {
  nationView: {
    active: {
      'fill': 'white',
      'stroke': 'black',
      'stroke-width': 1.5
    },
    inactive: {
      'fill': 'white',
      'stroke': 'black',
      'stroke-width': 1.0
    }
  },
  stateView: {
    active: {
      'fill': '#DCDCDC',
      'stroke': 'black',
      'stroke-width': 1.5
    },
    inactive: {
      'fill': 'transparent',
      'stroke': 'black',
      'stroke-width': 1.0
    }
  }
};
+  if (category == "total") { return "#2678b2"; }
 +  else if (category == "thermoelectric") { return "#edc948"; }
 +  else if (category == "publicsupply") { return "#76b7b2"; }
 +  else if (category == "irrigation") { return "#59a14f"; }
 +  else if (category == "industrial") { return "#e15759"; }
 +  else if (category == "other") { return "#A9A9A9"; }
```

Which gives these colors:
![image](https://user-images.githubusercontent.com/12039957/38375672-99b31390-38ab-11e8-84d7-1cf6007b2e0f.png)

Which, since they're in the vizzy, are not just prototyped but are already in action.

**Comment by aappling-usgs** [comment]
@jread-usgs how long did this one take you?

**Comment by jread-usgs** [comment]
I did this (choose colors) as part of #31, so the hours are wrapped up in what I have as a running tally for that issue, which is currently 6 hours. This part may have been 30mins~1hr
